.github/workflows/assign-and-label.yml

```diff
@@ -0,0 +1,26 @@
+name: "Auto Assign"
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - "release/**"
+    types: [opened]
+
+jobs:
+  add-reviewers:
+    name: "Auto assign reviewers"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kentaro-m/auto-assign-action@v1.1.0
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  pr-labeler:
+    name: "Auto label the Pull Reqest"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler-action@v3
+        with:
+          configuration-path: .github/pr-labeler.yml # optional, .github/pr-labeler.yml is the default value
```

Comment:
if optional && it's already the default value, do we need this line?

Reply:
removed